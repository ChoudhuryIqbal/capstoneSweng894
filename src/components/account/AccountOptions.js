import React from 'react'
import { View } from 'react-native'
import { ButtonCustom } from '../common/ButtonCustom' 


/**
 * Custom button for Account options
 * 
 * @param {} props 
 */
const AccountOptions = (props) => {
    return (
        <View style={props.viewStyle}>
            <ButtonCustom
                onPress={props.onPress}
                buttonStyle={props.buttonStyle}
                textStyle={props.textStyle}
            >
                {props.children}
            </ButtonCustom>
        </View>
    )
}

export default AccountOptions