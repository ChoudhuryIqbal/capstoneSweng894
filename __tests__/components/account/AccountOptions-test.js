import React from 'react'
import AccountOptions from '../../../src/components/account/AccountOptions'
import {create} from 'react-test-renderer';

describe('Account Image correctly renders', () => {
    test('Matches the snapshot', () => {
        const accountOption = create(
            <AccountOptions
                onPress={() => {}}
                buttonStyle={{}}
                textStyle={{}}
                viewStyle={{}}
                children={'Test'}
            />  
        )
        expect(accountOption.toJSON()).toMatchSnapshot();
    })
})