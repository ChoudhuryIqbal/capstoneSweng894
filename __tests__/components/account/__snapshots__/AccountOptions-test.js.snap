// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Image correctly renders Matches the snapshot 1`] = `
<View
  style={Object {}}
>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={Object {}}
    >
      Test
    </Text>
  </View>
</View>
`;
